tests/components/rtsptowebrtc/test_config_flow.py

```diff
@@ -0,0 +1,118 @@
+"""Test the RTSPtoWebRTC config flow."""
+
+from __future__ import annotations
+
+from typing import Any
+from unittest.mock import patch
+
+import rtsp_to_webrtc
+
+from homeassistant import config_entries, setup
+from homeassistant.components.rtsptowebrtc import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+CONFIG: dict[str, Any] = {}
+
+
+async def test_web_full_flow(hass: HomeAssistant):
+    """Check full flow."""
+    assert await setup.async_setup_component(hass, DOMAIN, CONFIG)
```

Comment:
Why do we do this?

Reply:
Ah, i think i didn't realize these were not needed. My primary introduction to home assistant has been via nest which is fairly legacy (and as you know i've been wanting to move away from configuration yaml) -- Removed.